app/src/main/java/emu/skyline/preference/ExportCustomSettingsPreference.kt

```diff
@@ -48,15 +48,16 @@ class ExportCustomSettingsPreference @JvmOverloads constructor(context : Context
                 - Docked: $systemIsDocked
                 
                 GPU
-                - Driver: $gpuDriver (executors: $gpuExecSlotCount slots, threshold of $gpuExecFlushThreshold)
+                - Driver: $gpuDriver
+                - Executors: $gpuExecSlotCount slots, threshold of $gpuExecFlushThreshold)
```

Comment:
There is no `(` to go along with the `)`

Reply:
fixed!